R/ebook.R

```diff
@@ -33,7 +33,7 @@ ebook <- function() {
     msg = "Use style: Vlaanderen when the language is nl"
   )
   assert_that(
-    style == "Flanders" || lang != "en",
+    style == "Flanders" || !lang %in% c("en", "fr"),
```

Comment:
I think we can simply this to 
```suggestion
    style == "Flanders" || lang == "nl",
```